src/test/java/org/jenkinsci/plugins/workflow/support/steps/ExecutorStepTest.java

```diff
@@ -400,6 +402,41 @@ private void startJnlpProc() throws Exception {
         });
     }
 
+    @Test public void detailsExported() throws Exception {
+        story.addStep(new Statement() {
+            @Override
+            public void evaluate() throws Throwable {
+                DumbSlave s = story.j.createOnlineSlave();
+
+                WorkflowJob p = story.j.jenkins.createProject(WorkflowJob.class, "demo");
+                p.setDefinition(new CpsFlowDefinition(
+                        "node('" + s.getNodeName() + "') {\n" +
+                                "    sleep 10\n" +
+                                "}"));
+
+                WorkflowRun b = p.scheduleBuild2(0).waitForStart();
+                Thread.sleep(100);
+                JenkinsRule.WebClient wc = story.j.createWebClient();
+                Page page = wc
+                        .goTo("computer/" + s.getNodeName()
+                                        + "/api/xml?depth=2&xpath=//currentExecutable/number/text()",
+                                "text/plain");
+
+                assertEquals("1", page.getWebResponse().getContentAsString());
+                page = wc.goTo("computer/" + s.getNodeName()
+                                + "/api/xml?depth=2&xpath=//currentExecutable/displayName/text()",
+                        "text/plain");
+                assertEquals("part of " + p.getDisplayName() + " #1",
```

Comment:
Test would run more quickly if you get all three fields in one response (easy enough with `tree`).